README.rst

```diff
@@ -220,3 +220,10 @@ Contributing
 ------------
 
 Please see our `contributing guide <https://github.com/mne-tools/mne-bids/blob/master/CONTRIBUTING.md>`_.
+
+MEG, EEG, iEEG Data From Unsupported Acquisition Systems
+--------------------------------------------------------
+
+Currently, we support the file formats that are in the BIDS specification, and certain manufacturer
```

Comment:
In what way do we support them? Only reading?

Reply:
Support reading in mne-python means we can support `write_raw_bids`. I updated the explanatino. LMK WYT.

Reply:
👍